tensorlayer/layers/recurrent.py

```diff
@@ -558,7 +560,7 @@ def __init__(
     @property
     def state_size(self):
         """State size of the LSTMStateTuple."""
-        return (LSTMStateTuple(self._num_units, self._num_units) if self._state_is_tuple else 2 * self._num_units)
+        return LSTMStateTuple(self._num_units, self._num_units) if self._state_is_tuple else (2 * self._num_units)
```

Comment:
are you sure it is the same?

Reply:
to me it really looks like redundant parentesis... I will revert this change just in doubt ;)
Not a big deal